src/BoilerPlate/Game.cpp

```diff
@@ -10,9 +10,10 @@ Game::Game(float currentHeight, float currentWidth)
 	player = Player();
 	player.UpdateFrameSize(height, width);
 	ResetLimiter();
-	asteroidCount = 5;
+	asteroidCount = 4;
```

Comment:
Move this to a constant. Avoid using magic numbers, only use literals when you're initializing something to a default value irrelevant to game logic.